src/main/java/in/twizmwaz/cardinal/module/modules/startTimer/StartTimer.java

```diff
@@ -62,16 +63,23 @@ public void run() {
             }
             if (time <= 60 && time >= 20 && time % 20 == 0) {
                 for (Player player : Bukkit.getOnlinePlayers()) {
-                    if (Settings.getSettingByName("Sounds") != null && Settings.getSettingByName("Sounds").getValueByPlayer(player).getValue().equalsIgnoreCase("on")) {
+                    if (Settings.getSettingByName("Sounds") != null && Settings.getSettingByName("Sounds").getValueByPlayer(player).getValue().equalsIgnoreCase("on") && !Teams.getTeamByPlayer(player).get().isObserver()) {
                         player.playSound(player.getLocation(), Sound.NOTE_PLING, 1, 1);
                     }
+                    if (!Teams.getTeamByPlayer(player).get().isObserver()) {
+                        player.showTitle(new TextComponent(ChatColor.YELLOW + "" + (time/20)), new TextComponent(""), 0, 10, 10);
+                    }
                 }
             }
             if (time == 0) {
                 for (Player player : Bukkit.getOnlinePlayers()) {
-                    if (Settings.getSettingByName("Sounds") != null && Settings.getSettingByName("Sounds").getValueByPlayer(player).getValue().equalsIgnoreCase("on")) {
+                    if (Settings.getSettingByName("Sounds") != null && Settings.getSettingByName("Sounds").getValueByPlayer(player).getValue().equalsIgnoreCase("on") && !Teams.getTeamByPlayer(player).get().isObserver()) {
                         player.playSound(player.getLocation(), Sound.NOTE_PLING, 1, 2);
                     }
+                    if (!Teams.getTeamByPlayer(player).get().isObserver()) {
+                        String title = new LocalizedChatMessage(ChatConstant.UI_MATCH_START_TITLE).getMessage(player.getLocale());
+                        player.showTitle(new TextComponent(ChatColor.GREEN + title), new TextComponent(""), 0, 10, 10);
+                    }
                 }
             }
```

Comment:
The countdown is already coded with the 3,2 and 1 sounds, and the "Go!" message is localized with the time == 0
also i wouldn't "for all players -> if count =0" it makes more sense "if count = 0. for all players"